start.py

```diff
@@ -1129,10 +1164,19 @@ def _setup_ssl_encryption_authentication(cluster, service):
     cluster.primary_node.execute(' && '.join(ssl_authentication_commands + ssl_commands))
 
 
+def _get_navigator_stage_lib_name(cm_version, additional_stage_libs_version=None):
+    # Returns version as a tuple (e.g. ('5', '15', '0')).
```

Comment:
Can probably remove this comment (obvious from the code what the next line does :-p)

Reply:
Done.